lib/mayaUsd/ufe/UsdTransform3d.cpp

```diff
@@ -118,22 +128,73 @@ Ufe::Vector3d UsdTransform3d::translation() const
 	return Ufe::Vector3d(x, y, z);
 }
 
-Ufe::RotateUndoableCommand::Ptr UsdTransform3d::rotateCmd()
+#if UFE_PREVIEW_VERSION_NUM >= 2013
+Ufe::Vector3d UsdTransform3d::rotation() const
```

Comment:
With UFE < 0.2.13, you could query translation, but not rotate or scale.  Oversight fixed in UFE, implemented here in maya-usd.